src/MenuItem/AsciiArtItem.php

```diff
@@ -64,19 +66,16 @@ public function getRows(MenuStyle $style, bool $selected = false) : array
 
             switch ($this->position) {
                 case self::POSITION_LEFT:
-                    return $row;
                     break;
                 case self::POSITION_RIGHT:
-                    $row = rtrim($row);
-                    $padding = $padding - ($this->artLength - mb_strlen($row));
+                    $padding -= ($this->artLength - $length);
```

Comment:
Those can be removed. The padding is simply equal to
```php
$padding = $style->getContentWidth() - $this->artLength;
```
and can be moved out of the array_map (line 65).

Moreover the $length doesn't need to be calculated (you made everything so it's based on the longest row) so you can remove line 63 too (can't comment on it).

Reply:
Nice. works perfect.